Tasks/Common/webdeployment-common-v2/ziputility.ts

```diff
@@ -25,13 +48,32 @@ const extractUsingUnzip = async (fromFile: string, toDir: string) => {
         .exec();
 }
 
+const isMSBuildPackageZip = async (packageZipPath: string): Promise<boolean> => {
+    let pacakgeComponent = await getArchivedEntries(packageZipPath);
```

Comment:
where is this getArchivedEntries function defined ? will this return all entries in the zip ? what happens if the zip is too large.